tests/common_name.py

```diff
@@ -1,28 +0,0 @@
-# -*- coding: utf-8 -*-
-import pytest
-
-from esteid.helpers import get_name_from_legacy_common_name, get_id_from_legacy_common_name
```

Comment:
As these methods still exist I believe the tests should also remain.

Reply:
Fair, got them back.